### PR TITLE
Sharihovey is adding a file for Nashua NH

### LINKED ISSUE #150: Add a file for sharihovey hometown
# Adding file for NashuaNH

- [x] Create a branch

- [x] test

- [x] Go home

- [x] Pick up kids

- [x] Walk dog

- [x] Make dinner

### PR DESCRIPTION
**Pull request by sharihovey**
This PR resolves #150 

@michelp70 what do you like to eat when you travel?

### CODE REVIEW

File: sharihovey-nashua.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+##Nashua NH
+
+### Great places to eat
```
Comment (sharihovey) [inline sharihovey-nashua.md]: @michelp70 Stella Blue is a great place to eat in down town Nashua.

File: sharihovey-nashua.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+##Nashua NH
```
Comment (ChandlerLutz) [inline sharihovey-nashua.md]: @sharihovey looks good!! Any good pizza places?

**Comment by michelp70** [comment]
#LGTM

File: sharihovey-nashua.md (hunk 2 above)
Comment (sharihovey) [inline sharihovey-nashua.md]: Hmm..Maybe Sal's, oh Whole Foods makes great pizza.

**Pull request merged**